src/components/NestedAccordionMenu/index.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import List from "@material-ui/core/List";
+import { makeStyles } from "@material-ui/core/styles";
+import Menu from "./Menu";
+
+export default function NestedList(props) {
```

Comment:
It seems like you could put the Menu component directly in here since NestedList is only rendering Menu.